### PR TITLE
Add disabled button function

### PR DESCRIPTION
**Pull request by Haroonchik**
Added the function to disable the button and added the disabled property to the home-page.css file

### CODE REVIEW

**Comment by HarshKapadia2** [comment]
Thank you for the PR!
You've handled the issue well 👍 

Just one change. In `./templates/home-page.html`, could you please include the script in the head block with a `defer` attribute?
Example:
```html
{% block head %}
	<script defer src="{{ url_for('static', filename='js/home-page.js') }}"></script>
{% endblock %}
```

**Comment by Haroonchik** [comment]
@HarshKapadia2 Ready. You can please add to the PR that you have approved add label: hacktoberfest-accepted. You can see how to do this here: https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Comment by HarshKapadia2** [comment]
> You can please add to the PR that you have approved add label: hacktoberfest-accepted. You can see how to do this here: hacktoberfest.digitalocean.com/hacktoberfest-update

I have already met this criteria.
The repo has a `hacktoberfest` topic and once I merge your PR, it will be valid for Hacktoberfest 2020.
Don't worry.

File: templates/home-page.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% extends 'base.html' %}
 
 {% block head %}
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/home-page.css') }}">
+	<link defer rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/home-page.css') }}">
```
Comment (HarshKapadia2) [inline templates/home-page.html]: This line was never to be changed, @Haroonchik. Please let the original line be.

The `script` tag needs to be included below this line with an added `defer` attribute to it.

**Comment by Haroonchik** [comment]
@HarshKapadia2  Ok😁

File: templates/home-page.html (hunk 1 above)
Comment (Haroonchik) [reply templates/home-page.html]: Ohhh, sry. I'll fix it now.

**Comment by Haroonchik** [comment]
@HarshKapadia2 Hastened in that commit and was wrong. Now everything is all right.

File: templates/home-page.html [hunk 2]
```diff
@@ -29,5 +29,5 @@ <h3>{{ message }}</h3>
 
 		<div>NOTE: The prediction will take around 20 seconds, so please be patient after clicking the 'Predict' button.</div>
 	</main>
-	<script src="{{ url_for('static', filename='js/home-page.js') }}"></script>
+	<script defer src="{{ url_for('static', filename='js/home-page.js') }}"></script>
```
Comment (HarshKapadia2) [inline templates/home-page.html]: Please add this line to the head block.

**Comment by Haroonchik** [comment]
@HarshKapadia2 I did it )

**Pull request merged**